src/Microsoft.AspNetCore.Server.Kestrel/KestrelServer.cs

```diff
@@ -175,7 +175,7 @@ public void Start<TContext>(IHttpApplication<TContext> application)
                             }
                             else
                             {
-                                _logger.LogWarning(0, ex, $"Unable to bind to {parsedAddress.ToString()} on the IPv6 loopback interface.");
```

Comment:
I think the scariest part of the log message is the stack trace. How about logging the exception message only?